tests/unit/FilingHistoryList.spec.ts

```diff
@@ -700,8 +700,32 @@ describe('Filing History List - redirections', () => {
     await flushPromises()
 
     // verify redirection
+    // const accountId = JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id
+    // const createUrl = 'https://edit.url/BC1234567/correction/?correction-id=110514'
+    // expect(window.location.assign).toHaveBeenCalledWith(createUrl + '&accountid=' + accountId)
```

Comment:
Can this be deleted?

Or is there a reason it should remain? (if so, add comment)

Reply:
I forgot to delete them. My bad 😞